src/connector/src/parser/protobuf/parser.rs

```diff
@@ -729,4 +744,55 @@ mod test {
             example_oneof: Some(ExampleOneof::OneofInt32(123)),
         }
     }
+
+    #[tokio::test]
+    async fn test_any_schema() -> Result<()> {
+        let location = schema_dir() + "/any-schema.pb";
+        println!("location: {}", location);
+        let message_name = "test.TestAny";
+        let info = StreamSourceInfo {
+            proto_message_name: message_name.to_string(),
+            row_schema_location: location.to_string(),
+            use_schema_registry: false,
+            ..Default::default()
+        };
+        let parser_config = SpecificParserConfig::new(
+            SourceStruct::new(SourceFormat::Plain, SourceEncode::Protobuf),
+            &info,
+            &HashMap::new(),
+        )?;
+        let conf = ProtobufParserConfig::new(parser_config.encoding_config).await?;
+        let c = conf.map_to_columns().unwrap();
+
+        println!("Current column: {:?}", c);
+
+        // let value = DynamicMessage::decode(conf.message_descriptor, PRE_GEN_PROTO_DATA).unwrap();
+
+        // assert_eq!(
+        //     value.get_field_by_name("id").unwrap().into_owned(),
+        //     Value::I32(123)
+        // );
+        // assert_eq!(
+        //     value.get_field_by_name("address").unwrap().into_owned(),
+        //     Value::String("test address".to_string())
+        // );
+        // assert_eq!(
+        //     value.get_field_by_name("city").unwrap().into_owned(),
+        //     Value::String("test city".to_string())
+        // );
+        // assert_eq!(
+        //     value.get_field_by_name("zipcode").unwrap().into_owned(),
+        //     Value::I64(456)
+        // );
+        // assert_eq!(
+        //     value.get_field_by_name("rate").unwrap().into_owned(),
+        //     Value::F32(1.2345)
+        // );
+        // assert_eq!(
+        //     value.get_field_by_name("date").unwrap().into_owned(),
+        //     Value::String("2021-01-01".to_string())
+        // );
```

Comment:
Should these lines be uncommented?

Reply:
The tests here haven't been fully tested and ensured, thus I temporarily commented these 🤣